elements/pfe-navigation/src/pfe-navigation.html

```diff
@@ -0,0 +1,66 @@
+<div class="pfe-navigation__wrapper">
+  <div class="pfe-navigation__container">
+    <div class="pfe-navigation__logo">
+      <slot name="logo"></slot>
+    </div>
+    <div class="pfe-navigation__skip">
+      <slot name="skip"></slot>
+    </div>
+    <div class="pfe-navigation__main">
+      <slot></slot>
+    </div>
+    <div class="pfe-navigation__utility">
+      <pfe-navigation-item pfe-icon="menu">
+        <h2 slot="trigger">Menu</h2>
+        <div slot="tray" hidden>
+          <div class="pfe-navigation__utility--mobile">
+            <div class="pfe-navigation__utility--mobile--top">
+              <slot name="mobile-search"></slot>
+            </div>
+            <slot name="mobile-menu"></slot>
```

Comment:
I don't see this slot referenced in the schema file.

Reply:
It's not a slot for users, it's a slot to inject content into dynamically during navigation initialization.  Only the web component puts stuff in there.